### ISSUE TITLE
Phonegap Build Gradle error 

### ISSUE THREAD

**Issue by jrm0695** [Wizcorp/phonegap-facebook-plugin#1145]
When compiling with the lastest version of PGB, I always have this error : "Oh geez. Your build failed. Sorry, but a problem occurred on the build server."; I've seen it's due to Gradle compilation but I can't move to Ant because AdMob, that I use, only works with Gradle... So I'm kind of stranded right now, I've to choose between putting ads on my app and enable the user to log in with his facebook account ?
I'm not used to work with cordova itself...

**Comment by bau720123** [comment]
+1

**Comment by jrm0695** [comment]
bau720123, do you know other alternative to get along with facebook with PGB ?

**Comment by bau720123** [comment]
hi @jrm0695 
I have a way in ant build
but if in gradle，no way for now

**Comment by jrm0695** [comment]
oh, and do you know if the Javascript API provided by Facebook works in PG apps ?

**Comment by jeduan** [comment]
I don't think this plugin is currently using gradle or doing anything to not support ant, so you might be misinterpreting the error.

**Comment by jrm0695** [comment]
You mean it's not actually an error ? Yeah but it looks there isn't other alternative to this plugin to make facebook account login in apps.. And I can't move from gradle because Admob uses it to work... I'm now trying the JS API provided by Facebook

**Comment by jrm0695** [comment]
Ok as I thought the JS API doesn't work .... jeduan I've seen you did a plugin to access Facebook from Phonegap, is it working with PGB ?

**Comment by jrm0695** [comment]
By the way, I've seen you made a very beautiful iOS-style preloader, can I use it for my app ? :)

**Comment by jeduan** [comment]
Preloader?

**Comment by dishcandanty** [comment]
+1
